llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -4188,28 +4191,8 @@ ElementCount LoopVectorizationCostModel::getMaximizedVFForTarget(
         llvm::bit_floor(WidestRegister.getKnownMinValue() / SmallestType),
         ComputeScalableMaxVF);
     MaxVectorElementCountMaxBW = MinVF(MaxVectorElementCountMaxBW, MaxSafeVF);
+    MaxVF = MaxVectorElementCountMaxBW;
```

Comment:
nit:
```suggestion
    MaxVF = MinVF(MaxVectorElementCountMaxBW, MaxSafeVF);
```
(MaxVectorElementCountMaxBW is not used after this point)